package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "topsort-events",
```

Comment:
Why does it say `topsort-events` here? I guess we need to run npm install again.